tests/jballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/freeze-and-isfrozen.bal

```diff
@@ -458,9 +459,9 @@ function testValidSelfReferencingValueFreeze() returns [string, boolean] {
 function testStructureWithErrorValueFreeze() returns boolean {
     string errReason = "test error";
     error e = error(errReason);
-    map<anydata|error> m = { err: e };
+    map<value:Cloneable|error> m = { err: e };
```

Comment:
Do we need this change?

Reply:
We need this right because we call `cloneReadOnly` on `m`

Reply:
Oh my bad let me check

Reply:
Looks like there is a bug. Trying to compare anydata with individual members of Cloneable.

Reply:
Fixed here cb4bb33acbe4a903ed0e4d94376089f63d7c26e3

Reply:
Did a quick fix may need to refactor the code.